.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - make travis
 
 before_script:
+  - go install ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
Since it is a kind of build dependency, maybe to move it to script section?
```
script:
  - go install ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
  - make dep-check
  - yamllint -c .yamllint.yml $(git ls-files '*.yaml' '*.yml' | grep -v 'vendor/')
  - make all && ./scripts/travis-integration-tests.sh minikube
```

Reply:
Sure, that's fine!